src/app/header/header.component.css

```diff
@@ -152,9 +152,14 @@
 }
 
 .settings-gear-dark-mode {
+    margin-right: 2.3rem;
     color: var(--dark-mode-anti-theme);
 }
 
+.settings-gear-dark-mode:hover {
+    background-color: #545454 !important;
```

Comment:
Do we already have this color used somewhere? What does the off-canvas menu button use?

Reply:
its the same color

Reply:
If it's fine with you, maybe we can create a variable for this?

Reply:
sure ill go ahead and create one right now